util/users.py

```diff
@@ -55,3 +55,21 @@ async def delete_user(identifier: user_models.UserIdentifier) -> None:
     if response.deleted_count == 0:
         detail = "User not found and could not be deleted"
         raise exceptions.UserNotFoundException(detail=detail)
+
+
+async def validate_user_info(
```

Comment:
should be called something along the lines of `try_to_login_user` or `attempt_user_login` or something that tells what the function does, even if it is more verbose.